rpcs3/Emu/RSX/Common/ProgramStateCache.h

```diff
@@ -214,16 +214,20 @@ class program_state_cache
 
 			lock.upgrade();
 			auto [it, inserted] = m_fragment_shader_cache.try_emplace(rsx_fp);
-			new_shader = &(it->second);
-			recompile = inserted;
+			new_shader          = &(it->second);
```

Comment:
Unnecessary whitespace change. clang-format? Remove it as you did not actually alter what is happening here.

Reply:
Yup, auto formatting. Reverted.